src/stylesheets/base/_typography.scss

```diff
@@ -29,6 +29,18 @@
     letter-spacing: none;
   }
 }
+
+.a-sup {
+  @include text($primary-font, normal, 1rem, $color-orange-red);
+
+  &:hover {
+    font-weight: bold;
+  }
+  @media (min-width: 550px) {
+    font-size: 1rem;
+  }
```

Comment:
this hasn't changed from the previous breakpoint. Need this?

Reply:
deleted. resolved in pr#4